src/java/org/apache/cassandra/db/guardrails/Guardrails.java

```diff
@@ -148,6 +148,13 @@
     new DisableFlag(state -> !CONFIG_PROVIDER.getOrCreate(state).getReadBeforeWriteListOperationsEnabled(),
                     "List operation requiring read before write");
 
+    /**
+     * Guardrail disabling ALLOW FILTERING statement within a query
+     */
+    public static final DisableFlag allowFilteringEnabled =
+    new DisableFlag(state -> !CONFIG_PROVIDER.getOrCreate(state).getAllowFilteringEnabled(),
```

Comment:
This is kind of a `Guardrails.java` problem, not a `your patch` problem ;), but I think the rhs of assignment on a new line should be indented one block.

Reply:
Yeah - in general I found the alignment jarring and inconsistent w/expectations. One indent in should be fine and we can do it full file on a single commit so it's easy to spot check on review.